fdbclient/include/fdbclient/GenericManagementAPI.actor.h

```diff
@@ -485,6 +486,11 @@ Future<ConfigurationResult> changeConfig(Reference<DB> db, std::map<std::string,
 							return ConfigurationResult::DATABASE_IS_REGISTERED;
 						}
 					}
+
+					if (newConfig.encryptionAtRestMode != oldConfig.encryptionAtRestMode) {
```

Comment:
I notice that this block can be bypassed if the user specifies the `force` parameter. We should probably not allow that (and I think we need to make a similar change for the tenant mode).